### PR TITLE
update non-helm migration steps to only what is needed

### PR DESCRIPTION
**Pull request by perk-sumo**
###### Description

Backport #767 to release-v1.0

###### Testing performed

- [ ] ci/build.sh
- [ ] Redeploy fluentd and fluentd-events pods
- [ ] Confirm events, logs, and metrics are coming in

### CODE REVIEW

File: deploy/docs/v1_migration_doc.md [hunk 1]
```diff
@@ -165,8 +165,15 @@ above the output plugin section [here](https://github.com/SumoLogic/sumologic-ku
...
-#### 1. Tear down existing Fluentd, Prometheus, Fluent Bit resources
-Run the commands mentioned [here](https://github.com/SumoLogic/sumologic-kubernetes-collection/blob/master/deploy/docs/Non_Helm_Installation.md#tear-down) to delete the respective resources.
+#### 1. Tear down existing Fluentd, Prometheus, Fluent Bit and Falco resources
+You will need the YAML files you created when you first installed collection. Run the following commands to remove Falco, Fluent-bit, Prometheus Operator and FluentD.  You do not need to delete the Namespace and Secret you originally created as they will still be used.
+
+```sh
+kubectl delete -f falco.yaml
+kubectl delete -f fluent-bit.yaml
+kubectl delete -f prometheus.yaml
+kubectl delete -f fluentd-sumologic.yaml
```
Comment (sumo-drosiek) [inline deploy/docs/v1_migration_doc.md:175]: Should we care about the Prometheus CRDs?

Comment (frankreno) [reply deploy/docs/v1_migration_doc.md:175]: Ahh good point.  If we dont, we get the CRD's already exist error.

**Comment by pmalek-sumo** [comment]
Do we still want to backport this or perhaps it outdated itself already? @perk-sumo

File: deploy/docs/v1_migration_doc.md (hunk 1 above)
Comment (perk-sumo) [reply deploy/docs/v1_migration_doc.md:176]: Is that still true? On my 1.18 cluster I get the following:
```
$ kubectl delete -f https://raw.githubusercontent.com/coreos/prometheus-operator/release-0.38/example/prometheus-operator-crd/monitoring.coreos.com_podmonitors.yaml
customresourcedefinition.apiextensions.k8s.io "podmonitors.monitoring.coreos.com" deleted

$ kubectl apply -f https://raw.githubusercontent.com/coreos/prometheus-operator/release-0.38/example/prometheus-operator-crd/monitoring.coreos.com_podmonitors.yaml
customresourcedefinition.apiextensions.k8s.io/podmonitors.monitoring.coreos.com created

$ kubectl apply -f https://raw.githubusercontent.com/coreos/prometheus-operator/release-0.38/example/prometheus-operator-crd/monitoring.coreos.com_podmonitors.yaml
customresourcedefinition.apiextensions.k8s.io/podmonitors.monitoring.coreos.com configured

$ kubectl apply -f https://raw.githubusercontent.com/coreos/prometheus-operator/release-0.38/example/prometheus-operator-crd/monitoring.coreos.com_podmonitors.yaml
customresourcedefinition.apiextensions.k8s.io/podmonitors.monitoring.coreos.com configured
```

Comment (pmalek-sumo) [reply deploy/docs/v1_migration_doc.md:176]: When I was testing `kube-prometheus-stack` version bump PR https://github.com/SumoLogic/sumologic-kubernetes-collection/pull/1089 I remember also being able to just apply new yamls on top of the old ones without an error.

Comment (perk-sumo) [reply deploy/docs/v1_migration_doc.md:176]: I've just checked that with EKS 1.14.9 (the earliest we support) and kubectl version `v1.15.11` and `v1.19.3` and I see `configured` all the time.

Comment (sumo-drosiek) [reply deploy/docs/v1_migration_doc.md:176]: That's fine until we don't use `kubectl create -f https://raw.githubusercontent.com/coreos/prometheus-operator/release-0.38/example/prometheus-operator-crd/monitoring.coreos.com_podmonitors.yaml`

**Review by sumo-drosiek — APPROVED** [review]

File: deploy/docs/v1_migration_doc.md (hunk 1 above)
Comment (perk-sumo) [reply deploy/docs/v1_migration_doc.md:176]: 👍  thanks!
As we use `kubectl apply` in our documentation I'm going to not add this info to documentation and just merge the PR.

**Pull request merged**